src/components/v2/MyAccount/index.ts

```diff
@@ -0,0 +1 @@
+export * from './MyAccount';
```

Comment:
Let's follow the codebase's best practices and move the component inside this file directly and delete `MyAccount.tsx`.